bot2-lcm-utils/python/CMakeLists.txt

```diff
@@ -3,5 +3,20 @@ find_package(PythonInterp REQUIRED)
 # install all python packages in the src/ directory
 pods_install_python_packages(${CMAKE_CURRENT_SOURCE_DIR}/src)
 
-# executable scripts:  script-name  python-module
-pods_install_python_script(bot-log2mat bot_log2mat.log_to_mat)
+if(TARGET ${LCM_NAMESPACE}lcm)
+  get_target_property(LCM_INCLUDE_DIR ${LCM_NAMESPACE}lcm
+    INTERFACE_INCLUDE_DIRECTORIES)
+  get_filename_component(LCM_PREFIX "${LCM_INCLUDE_DIR}" DIRECTORY)
```

Comment:
`INTERFACE_INCLUDE_DIRECTORIES` is or can be a list:
`  INTERFACE_INCLUDE_DIRECTORIES "/home/fbudin/Devel/lcm;/home/fbudin/Devel/lcm-build/lcm"`